test/utils/mem-store.js

```diff
@@ -64,32 +39,22 @@ class MemStore {
       toJSON: () => {
         return {
           data: value,
-          multihash: hash,
+          multihash: hash
         }
       }
     }
   }
 
   async get (key) {
-    // const data = this._store.get(key)
-    const data = this._store[key]
```

Comment:
Should this have been removed?

Reply:
It seems it wasn't used at all. `eslint` complained about it (`no-unused-vars` rule)